src/node/rpc/common_handler_registry.h

```diff
@@ -146,7 +145,7 @@ namespace ccf
         if ((nodes != nullptr) && (consensus != nullptr))
         {
           NodeId primary_id = consensus->primary();
-          auto current_term = consensus->get_view();
+          auto [current_term, _] = consensus->get_committed_txid();
```

Comment:
I'll revert this back to get_view(), I briefly thought we didn't need it anymore, but we do for signatures it turns out.